.goreleaser.yml

```diff
@@ -0,0 +1,42 @@
+project_name: servicegateway
+before:
+  hooks:
+    - go vet ./...
+    - go test ./...
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    binary: servicegateway
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - '^test:'
+dockers:
+  -
+    image_templates:
+      - quay.io/mittwald/servicegateway:latest
+      - quay.io/mittwald/servicegateway:v{{ .Major }}
+      - quay.io/mittwald/servicegateway:v{{ .Major }}.{{ .Minor }}
+      - quay.io/mittwald/servicegateway:{{ .Tag }}
+      - quay.io/mittwald/servicegateway:stable
+      - quay.io/mittwald/servicegateway:v{{ .Major }}-alpine
```

Comment:
Why bother with a dedicated `-alpine` tag, if the regular tag is already Alpine-based?